.github/workflows/push.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+    - *
+  pull_request:
+    branches: [ master ]
+  
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+      with:
+        install: true
+        version: latest
+        
+    - name: Set up QEMU
+      id: qemu
+      uses: docker/setup-qemu-action@v1
+      with:
+        image: tonistiigi/binfmt:latest
+        platforms: all
+  
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+ 
+    -
+      name: Login to GitHub Container Registry
+      uses: docker/login-action@v1 
+      with:
+        registry: ghcr.io
+        username: ${{ secrets.CR_USERNAME }}
```

Comment:
Should this actually be `${{ github.repository_owner }}` according to https://github.com/docker/login-action#github-container-registry 

Also, should I create the PAT and add the token as a secret named CR_PAT?
I already added organization level secrets for DOCKERHUB_USERNAME & DOCKERHUB_TOKEN

Reply:
nvm, I understand, it should be the user that created the secret, in this case my username.
I will add a secret for CR_USERNAME as well

Reply:
yep, that's why i switched it